src/olympia/scanners/admin.py

```diff
@@ -48,41 +51,38 @@ def _is_safe_url(url, request):
         urlparse(settings.EXTERNAL_SITE_URL).netloc,
     )
     require_https = request.is_secure() if request else False
-    return is_safe_url(url, allowed_hosts=allowed_hosts,
-                       require_https=require_https)
+    return is_safe_url(url, allowed_hosts=allowed_hosts, require_https=require_https)
 
 
 class PresenceFilter(SimpleListFilter):
     def choices(self, cl):
         for lookup, title in self.lookup_choices:
             yield {
-                'selected': self.value() == lookup,
-                'query_string': cl.get_query_string(
-                    {self.parameter_name: lookup}, []
-                ),
-                'display': title,
+                "selected": self.value() == lookup,
+                "query_string": cl.get_query_string({self.parameter_name: lookup}, []),
+                "display": title,
```

Comment:
the "switch back to single quotes " commit should have reverted these changes?  (I just picked a few files to check - I don't know how widespread it is)

Reply:
Yes, I am working on fixing that.

Reply:
I am going to re-apply black entirely to fix conflicts so this shuld be fixed.